common/src/main/kotlin/kiwiband/dnb/actors/creatures/status/CreatureStatus.kt

```diff
@@ -0,0 +1,59 @@
+package kiwiband.dnb.actors.creatures.status
+
+import kiwiband.dnb.math.MyMath
+import kotlin.math.min
+import kotlin.random.Random
+
+class CreatureStatus(
+    level: Int,
+    health: Int = -1,
+    experience: Int = 0
+) {
+
+    var maxHealth = 13 + 2 * level
```

Comment:
Какие комментарии? // max health is 13 + 2 * level just because
Это и правда магические формулы, какая-то бизнес логика, у которой нет каких-то серьезных предпосылок или оснований